core-bundle/src/Util/SymlinkUtil.php

```diff
@@ -59,13 +55,15 @@ public static function validateSymlink(string $target, string $link, string $roo
             throw new \InvalidArgumentException('The symlink path must not be empty.');
         }
 
+        $link = Path::normalize($link);
+
         if (false !== strpos($link, '../')) {
             throw new \InvalidArgumentException('The symlink path must not be relative.');
```

Comment:
Sidenote: from a technical standpoint it would not be of any problem if the symlink path would include a ('../') - I assume we do this to prevent escaping the project dir level?

Reply:
Windows does not support relative symlinks. (It actually does, but only in a very limited scope.)

Reply:
For windows `Filesystem#symlink` would receive a canonicalized path now, so the relative links would be converted:
```
/path/to/somewhere/../foo  -->  /path/to/foo
```

Reply:
Since we are not sure why the code is there, I would highly recommend to leave it as is, unless

* you are 100% sure that you are not breaking anything and
* the change is technically required.

Reply:
Sure, we need to keep it anyways because `validateSymlink()` is part of the public API...